setup.py

```diff
@@ -15,7 +15,7 @@
 def read(f):
     return open(os.path.join(os.path.dirname(__file__), f)).read().strip()
 
-install_requires = ['aiohttp>=0.20', 'jinja2>=2.7']
+install_requires = ['aiohttp>=0.20', 'jinja2>=2.9']
```

Comment:
Do we need to bump version here or should I add an explicit check within setup method that will enable the flag only in case Jinja version is higher than `2.9`. First option means that we kinda break backwards compatibility for newest release of `aiohttp_jinja2` with older `jinja2` versions

Reply:
I think it should be fine, for older versions of jinja2 one can use older version of aiohttpp-jinja2